micro_manager/adaptivity/adaptivity.py

```diff
@@ -36,25 +39,10 @@ def get_similarity_dists(self, dt: float, similarity_dists: np.ndarray, data: np
         _similarity_dists = np.copy(similarity_dists)
 
         if data.ndim == 1:
-            dim = 0
-        elif data.ndim == 2:
-            _, dim = data.shape
-
-        number_of_sims, _ = _similarity_dists.shape
-
-        for counter_1, id_1 in enumerate(range(number_of_sims)):
-            for counter_2, id_2 in enumerate(range(number_of_sims)):
-                data_diff = 0
-                if id_1 != id_2:
-                    if dim:
-                        for d in range(dim):
-                            data_diff += abs(data[counter_1, d] - data[counter_2, d])
-                    else:
-                        data_diff = abs(data[counter_1] - data[counter_2])
-
-                    _similarity_dists[id_1, id_2] += dt * data_diff
-                else:
-                    _similarity_dists[id_1, id_2] = 0
+            data = np.expand_dims(data, axis=1)
```

Comment:
Please write a comment here on what you are actually doing. Bigger description is also okay.